### PR TITLE
Fix the logging for pushdown filters

### PR DESCRIPTION
**Pull request by isha97**

### CODE REVIEW

File: bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/ReadSessionCreator.java [hunk 1]
```diff
@@ -90,10 +90,13 @@ public ReadSessionResponse create(
       TableId table, ImmutableList<String> selectedFields, Optional<String> filter) {
     Instant sessionPrepStartTime = Instant.now();
     TableInfo tableDetails = bigQueryClient.getTable(table);
-
     TableInfo actualTable = getActualTable(tableDetails, selectedFields, filter);
 
     BigQueryReadClient bigQueryReadClient = bigQueryReadClientFactory.getBigQueryReadClient();
+    log.info(
```
[the remark this answers is not in the document]
Comment (isha97) [reply bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/ReadSessionCreator.java:96]: Done

File: spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/direct/DirectBigQueryRelation.java [hunk 2]
```diff
@@ -111,14 +111,14 @@ public RDD<Row> buildScan(String[] requiredColumns) {
 
   @Override
   public RDD<Row> buildScan(String[] requiredColumns, Filter[] filters) {
+    compiledFilter = getCompiledFilter(filters);
     log.info(
         "|Querying table {}, parameters sent from Spark:"
             + "|requiredColumns=[{}],"
-            + "|filters=[{}]",
```
[the remark this answers is not in the document]
Comment (isha97) [reply spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/direct/DirectBigQueryRelation.java:117]: Done

**Review by davidrabinowitz — APPROVED** [review]

**Pull request merged**